src/pages/iou/IOUDetailsModal.js

```diff
@@ -132,9 +137,17 @@ class IOUDetailsModal extends Component {
         }
     }
 
+    // Finds if there is a reportAction pending for this IOU
+    findPendingAction() {
+        return _.find(this.props.reportActions, reportAction => reportAction.originalMessage
+            && Number(reportAction.originalMessage.IOUReportID) === Number(this.props.route.params.iouReportID)
+            && !_.isEmpty(reportAction.pendingAction));
+    }
+
     render() {
         const sessionEmail = lodashGet(this.props.session, 'email', null);
         const reportIsLoading = _.isUndefined(this.props.iouReport);
+        const pendingAction = this.findPendingAction();
```

Comment:
Bug:

- `findPendingAction` is returning a reportAction.
- `const pendingAction` holds a reportAction, not a string pendingAction string like 'add'
- We pass `pendingAction` to `<IOUPreview>` which is expecting an string, not a reportAction object.

Reply:
I'll put a small PR correcting it.

Reply:
Thanks

Reply:
A bit late, but I just remembered this. PR up: https://github.com/Expensify/App/pull/16019